### PR TITLE
remove-duplicate-vertices

### PR DESCRIPTION
**Pull request by ieskudero**
remove duplicated vertices after parsing a non indexed vertices array

### CODE REVIEW

**Comment by donmccurdy** [comment]
Could you explain further why you do not want duplicate vertices to be merged?

File: src/core/Geometry.js [hunk 1]
```diff
@@ -335,7 +335,8 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		}
 
 		this.computeFaceNormals();
-
+		this.mergeVertices();
+
```
Comment (Mugen87) [inline src/core/Geometry.js]: There seems to be trailing spaces. Please remove them otherwise the linter won't be happy :(

**Comment by WestLangley** [comment]
Actually, I don't think this is a good idea. Please see https://github.com/mrdoob/three.js/issues/14857#issuecomment-419232649.

**Comment by mrdoob** [comment]
@WestLangley Agreed.

**Comment by mrdoob** [comment]
I don't think we should change `Geometry` at this point.
We should, instead, focus on how to move away from it.

**Pull request closed without merging**